### ISSUE TITLE
Unable to move nodes into command window on multiplayer

### ISSUE THREAD

**Issue by zilvarwolf** [TeamDman/SuperFactoryManager#47]
FTB:Interactions 1.9.1 install, so 2.0.18.  Trying to build something in SFM and cannot create any nodes other than Group nodes in the command window while on a multiplayer server.  Clicking, dragging, right clicking, and profanity all have the same null effect.  On occasion, I've been able to get something like a flow node to show up but when I exit the SFM console and look later, it's almost always gone.

**Comment by TeamDman** [comment]
Sounds like an issue that was brought by using a machine placed before the update to field lengths. Since the NBT parser changed the machine might need to be broken and replaced in the world. Unless you've already tried that and are still experiencing the issue?

**Comment by zilvarwolf** [comment]
If that's the case, then I guess I'll have to wait until the next update to
Interactions, or update it manually.  There was no server update inbetween
my attempts to use the factory manager and the current version.  I have not
attempted to break and replace the block, but I can.

And I just tried that.  No change.  I did a break/replace and tried to drop
in a few nodes (2 triggers, 2 inputs, 2 outputs).  Nothing changed in the
command area.  Closed and reopened the interface and now it's hung.

very odd.
